http-proxy-configurator-operator/CONTRIBUTING.md

```diff
@@ -152,7 +153,7 @@ juju add-model charm-dev
 # Enable DEBUG logging
 juju model-config logging-config="<root>=INFO;unit=DEBUG"
 # Deploy the charm
-juju deploy ./<charm-name>.charm 
+juju deploy ./http-proxy-configurator_*.charm 
```

Comment:
Shouldn't this be http-proxy-configurator_amd64.charm o another filename resulting from packing?

Reply:
How about something like?
```suggestion
juju deploy ./http-proxy-configuration_$(dpkg --print-architecture).charm
```
Would that be valid?